server/athenian/api/__init__.py

```diff
@@ -43,7 +43,9 @@ class Formatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawTextHelpForm
   AUTH0_CLIENT_ID          Client ID of the Auth0 Machine-to-Machine Application
   AUTH0_CLIENT_SECRET      Client Secret of the Auth0 Machine-to-Machine Application
   ATHENIAN_INVITATION_KEY  Passphrase to encrypt the invitation links
-  """,
+  ATHENIAN_INVITATION_URL_PREFIX
+                           String with which any invitation URL starts, e.g. https://app.athenian.co/i/
+  """,  # noqa
```

Comment:
This is starting to look like a missed configuration file. We should consider putting these to a file in the next version.

Reply:
I agree with you, reading this configurations from a file will make things easier also from infra pov. Regarding the value, for now it should be `https://owl.athenian.co/i/`, for local development by default it should be `http://localhost:3000/i/`.

Reply:
nvm let's read it from env var.